### PR TITLE
if the user is not ready to install Alan, offer other options

### PR DESCRIPTION
**Pull request by depombo**

### CODE REVIEW

File: src/getting_started.md [hunk 1]
```diff
@@ -62,28 +65,19 @@ on app.start {
...
 
-which prints `Hello, World!`.
-
 You can also run the source file directly:
 
 ```bash
 alan hello_world.ln
 ```
 
-which will internally compile the code then run it, but you have to pay the compilation costs on each run.
```
Comment (dfellis) [inline src/getting_started.md:85]: I don't know about removing this line.

File: src/getting_started.md [hunk 2]
```diff
@@ -1,6 +1,9 @@
 # Getting Started
 
-This guide walks you through the steps needed to get the Alan compiler and runtimes on your local machine. To work on Alan itself, see [Contributing to Alan](./contributing.md). To learn more about Alan, see [Why we created Alan](https://alan-lang.org/why_alan.html).
+This guide walks you through the steps needed to get the Alan compiler and VM on your local machine. You can also:
+- Read through some idiomatic [Alan examples](./advanced_examples.md)
+- Read more about [Why we created Alan](https://alan-lang.org/why_alan.html)
+- [Contribute to Alan](./contributing.md)
```
Comment (dfellis) [inline src/getting_started.md:6]: Maybe a fourth bullet point on using the browser-based compiler and runtime on the homepage?

File: src/getting_started.md (hunk 1 above)
Comment (depombo) [reply src/getting_started.md:85]: It is not in the README https://github.com/alantech/alan#recommended-usage

File: src/getting_started.md (hunk 2 above)
Comment (depombo) [reply src/getting_started.md:6]: well, the playground is also available in the docs

File: src/getting_started.md (hunk 1 above)
Comment (depombo) [reply src/getting_started.md:85]: I will add it back until the compiler is faster in both places

Comment (dfellis) [reply src/getting_started.md:85]: That doesn't justify it's removal here. :)

Right now, the user may be wondering "why should I ever do the `alan compile ...` `alan run ...` dance?"

Comment (depombo) [reply src/getting_started.md:85]: It should be consistent.

**Review by dfellis — APPROVED** [review]

File: src/getting_started.md (hunk 1 above)
Comment (dfellis) [reply src/getting_started.md:85]: agreed

**Pull request merged**